### PR TITLE
Add set to convert gedmo annotations to attributes

### PR DESCRIPTION
**Pull request by acrobat**

### CODE REVIEW

**Comment by acrobat** [comment]
Are there any extra changes needed here?

File: config/sets/doctrine-gedmo-annotations-to-attributes.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```
Comment (samsonasik) [inline config/sets/doctrine-gedmo-annotations-to-attributes.php:9]: You need to use `RectorConfig` with `$rectorConfig->ruleWithConfiguration()` syntax

Comment (acrobat) [reply config/sets/doctrine-gedmo-annotations-to-attributes.php]: Fixed!

**Comment by acrobat** [comment]
The phpstan failure seem unrelated

**Comment by TomasVotruba** [comment]
I've fixed the PHPStan errors here: https://github.com/rectorphp/rector-doctrine/pull/120

Could you rebase?

**Comment by samsonasik** [comment]
please rebase

File: config/sets/doctrine-gedmo-annotations-to-attributes.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig
+        ->ruleWithConfiguration(AnnotationToAttributeRector::class, [
+            new AnnotationToAttribute('Gedmo\\Mapping\\Annotation\\Blameable'),
```
Comment (samsonasik) [inline config/sets/doctrine-gedmo-annotations-to-attributes.php:12]: double `\\` can be changed to single `\`

Comment (acrobat) [reply config/sets/doctrine-gedmo-annotations-to-attributes.php]: Fixed!

**Comment by acrobat** [comment]
Rebase done!

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**